diagnostic.md

```diff
@@ -32,23 +36,28 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+$("p").text()
+$("h2").html()
 
+$("p").text("hello")
+$("h2").html("good day")
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM is a representation of the document by the browser.  It allows us to communicate with api's and make changes to the document by making requests.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.